pkg/platform/kube/functionres/lazy.go

```diff
@@ -1553,8 +1554,18 @@ func (lc *lazyClient) generateCronTriggerCronJobSpec(functionLabels labels.Set,
 			eventBodyCurlArg)
 	}
 
+	// get cron job retries until failing a job (default=2)
+	jobBackoffLimit := int32(2)
```

Comment:
move to a resolve function and add a log line in case of failing parsing so we would know something happened

Reply:
Changed it to be received as int32 to save this parsing